std/experimental/allocator/building_blocks/stats_collector.d

```diff
@@ -664,15 +664,11 @@ unittest
     alloc.reallocate(b, 20);
     alloc.deallocate(b);
 
+    import std.file : deleteme;
     import std.stdio : File;
     import std.range : walkLength;
-    version(Posix)
-        auto f = "/tmp/dlang.std.experimental.allocator.stats_collector.txt";
-    version(Windows)
-    {
-        import std.process: environment;
-        auto f = environment.get("temp") ~ r"\dlang.std.experimental.allocator.stats_collector.txt";
-    }
+
+    auto f = deleteme ~ "dlang.std.experimental.allocator.stats_collector.txt";
```

Comment:
No `/tmp` on Android, use `std.file.deleteme` instead.  Does this clean up after itself or do I need to delete it too?